tests/unit/usecases/test_collect_nodes_statuses.py

```diff
@@ -20,6 +22,7 @@ def node_gateway(self):
         return MagicMock()
 
     @patch('usecases.collect_nodes_statuses.HATHOR_NODES', ['a', 'b'])
+    @freeze_time(datetime.now())
```

Comment:
Why do you need to freeze the time? I mean, you are not using a specific point in time in the tests.

Reply:
As we calculate `up_since` in runtime, it can change a second for two different instances of the same Node created in two different points of the code. It's an edge case that can bring intermittence to tests. So freeze the time will avoid it.